tests/functional/TestManyBricksVolume/run.sh

```diff
@@ -1,11 +1,8 @@
 #!/bin/bash
 
-CURRENT_DIR="$(pwd)"
-export HEKETI_SERVER_BUILD_DIR=../../..
-FUNCTIONAL_DIR="${CURRENT_DIR}/.."
-export HEKETI_SERVER="${FUNCTIONAL_DIR}/heketi-server"
+SCRIPT_DIR="$(cd "$(dirname "${0}")" && pwd)"
+FUNCTIONAL_DIR="${SCRIPT_DIR}/.."
 
-source "${FUNCTIONAL_DIR}/lib.sh"
+. "${FUNCTIONAL_DIR}/lib.sh"
```

Comment:
I am wondering why the change from "source" to "."
We use source throughout the code base.

Reply:
Well... I was trying to shorten those files. :-)